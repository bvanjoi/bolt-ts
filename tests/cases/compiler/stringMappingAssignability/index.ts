// From `github.com/microsoft/TypeScript/blob/v5.7.3/tests/cases/compiler/stringMappingAssignability.ts`, Apache-2.0 License

const x: Uppercase<string> = 42;
//~^ ERROR: Type 'number' is not assignable to type 'Uppercase'.
const y: Uppercase<string> = { foo: "bar" };
//~^ ERROR: Type '{ foo: string; }' is not assignable to type 'Uppercase'.
