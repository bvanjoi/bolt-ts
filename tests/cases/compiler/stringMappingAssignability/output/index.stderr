  × Type 'number' is not assignable to type 'Uppercase'.
   ╭─[../../tests/cases/compiler/stringMappingAssignability/index.ts:3:7]
 3 │ x
   · ─
   ╰────

  × Type '{ foo: string; }' is not assignable to type 'Uppercase'.
   ╭─[../../tests/cases/compiler/stringMappingAssignability/index.ts:5:7]
 5 │ y
   · ─
   ╰────
